### ISSUE TITLE
Git Master, not able to load TIF files. 

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#3832]
```c++
ofImage img;
img.load("0.tif");
```
and the console output
```
Assertion failed: (sizeof(uint64)==8), function TIFFClientOpen, file Source/LibTIFF4/./tif_open.c, line 99.
```

**Comment by kylemcdonald** [comment]
got this same result on OSX 10.10 with master, in 64-bit and 32-bit. likely related to the newly built FreeImage libraries.

**Comment by danoli3** [comment]
Yeah that's one bad assertion. (Especially with varying type sizes depending on platform).

Reported on FreeImage here: http://sourceforge.net/p/freeimage/bugs/214/

Potential fix posted in reply, will need to be adapted for other platforms.

**Comment by danoli3** [comment]
"/* The following are configuration checks. They should be redundant, but should not
	 * compile to any actual code in an optimised release build anyway. If any of them
	 * fail, (makefile-based or other) configuration is not correct */"

Oh wow, we forgot the "-DNDEBUG". I'll audit the makefile.

**Comment by danoli3** [comment]
Yeah, I think we missed the command. I'm going to make a PR with the script change, also going to build the library with the updated command and test to see if it fixes it.

**Comment by danoli3** [comment]
Confirmed fixed by updating libraries with NDEBUG (No Assertions).

**Comment by danoli3** [comment]
Pull request posted to fix this issue. @kylemcdonald

**Comment by bilderbuchi** [comment]
closed by #3847
